### DISCUSSION TITLE
WARN  [unocss] The labeled variant is experimental and may not follow semver.

### DISCUSSION THREAD

**Discussion by mehdicodecom** [unocss/unocss#3709] [Q&A]
I'm using unocss with my nuxt app, I'm getting this error when I run my app in terminal
I'm using these packages:
    "@unocss/nuxt": "^0.59.0",
    "@unocss/transformer-variant-group": "^0.59.0",
    
Error:
` WARN  [unocss] The labeled variant is experimental and may not follow semver.`
any suggestion why this error appears ?

**Comment by schelmo** [comment]
it is coming from labeled groups or peers like
`group/nav:focus-within`
